test/e2e/app-dir/actions/app/revalidate-multiple/page.js

```diff
@@ -24,16 +31,18 @@ export default async function Page() {
       <p>
         revalidate (tags: justputit): <span id="justputit">{data2}</span>
       </p>
+      <p>
+        revalidate (tags: yougotit): <span id="yougotit">{data3}</span>
+      </p>
       <form>
         <button
           id="revalidate"
           formAction={async () => {
             'use server'
-            revalidateTag('thankyounext')
-            revalidateTag('justputit')
+            revalidateTag('thankyounext', 'justputit')
```

Comment:
This fixture is not currently used by any other tests, but it seems fitting for this case, so I adapted it.